scripts/check_compliance.py

```diff
@@ -579,32 +579,47 @@ def run(self):
         report = ""
 
         whitelist_extensions =  ['.yaml', '.html']
-        whitelist_languages = ['CMake', 'HTML']
+        whitelist_languages = ['HTML']
+        blacklist_languages = ['CMake']
         with open('scancode.json', 'r') as json_fp:
             scancode_results = json.load(json_fp)
             for file in scancode_results['files']:
                 if file['type'] == 'directory':
                     continue
 
-                original_fp = str(file['path']).replace('scancode-files/', '')
+                orig_path = str(file['path']).replace('scancode-files/', '')
                 licenses = file['licenses']
-                if (file['is_script'] or file['is_source']) and (file['programming_language'] not in whitelist_languages) and (file['extension'] not in whitelist_extensions):
+                lang = file.get("programming_language", None)
+                extension = file.get("extension")
+                file_type = file.get("file_type")
+                kconfig = "Kconfig" in orig_path and file_type in ['ASCII text']
+                checked_file = (
+                        file.get("is_script") or
+                        file.get("is_source") or
```

Comment:
If `is_script` or `is_source` is True for a file, then `blacklist_languages` and `whitelist_languages` won't matter. Is that deliberate?